Traceback (most recent call last):
  File "svm.py", line 5, in <module>
    trX = np.load("../ch_tests/trX.npy")
  File "/usr/lib64/python2.6/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../ch_tests/trX.npy'
